test/Microsoft.TestPlatform.Common.UnitTests/ExtensionFramework/Utilities/RunSettingsProviderExtensionsTests.cs

```diff
@@ -156,6 +150,18 @@ public void UpdateTetsRunParameterSettingsNodeShouldOverrideValueIfKeyIsAlreadyP
             Assert.AreEqual(runSettingsWithTestRunParametersOverrode, this.runSettingsProvider.ActiveRunSettings.SettingsXml);
         }
 
+        [TestMethod]
+        public void UpdateTestRunParameterSettingsNodeWithSpecialCharactersNameShouldSucceed()
+        {
+            this.CheckRunSettingsAreUpdated("weburl:name", @"http://localhost//abc");
+        }
+
+        [TestMethod]
+        public void UpdateTestRunParameterSettingsNodeWithSingleCharacterNameShouldSucceed()
```

Comment:
Name this `TestRunParameterSettingsNodeCanBeJustASingleCharacter` ?

Reply:
Done.